docs/pages/pmd/devdocs/contributing.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
I think, we can delete this whole file. I didn't see any other internal link to this page except for the sidebar (which you already adjusted).